env/requirements_pytorch.yml

```diff
@@ -0,0 +1,18 @@
+name: data-science-cop-torch
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - python==3.8.13
```

Comment:
I think it would be better just to pn the major and minor python version here, rather than the build / bug fix number as well. so only 3.8, rather than 3.8.13. I believe there should be not conflicts using different versions of 3.8.X.

Reply:
[07231ce](https://github.com/MetOffice/data_science_cop/pull/29/commits/07231cebeca3944bf247e1be7dd8d987cfc1c8f5) Updated